.github/workflows/build.yml

```diff
@@ -142,7 +144,9 @@ jobs:
 
   build-dockers:
     runs-on: ubuntu-latest
-    needs: [build, build-arm]
+    needs: [build]
```

Comment:
I guess this will break if build_arm is slower than build and the artifact might not be available.